spark/src/main/scala/ai/chronon/spark/JoinBase.scala

```diff
@@ -188,14 +188,11 @@ abstract class JoinBase(joinConf: api.Join,
             val leftUnfilledRange = unfilledRange.shift(-shiftDays)
             val prunedLeft = leftDf.flatMap(_.prunePartitions(leftUnfilledRange))
             val filledDf =
-              computeJoinPart(prunedLeft, joinPart, joinLevelBloomMapOpt, smallMode)
+              computeJoinPart(prunedLeft, joinPart, bloomMapOpt, smallMode)
```

Comment:
nit: I don't think we need to pass in smallMode anymore? smallMode and bloomMapOpt are always mutually exclusive